packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -265,14 +196,14 @@ gmaps.InfoWindowOptions _infoWindowOptionsFromMarker(Marker marker) {
   if (marker.infoWindow.title?.isNotEmpty ?? false) {
     final HtmlElement title = HeadingElement.h3()
       ..className = 'infowindow-title'
-      ..innerText = marker.infoWindow.title;
+      ..innerText = marker.infoWindow.title!;
```

Comment:
Totally optional nit: could be nice to extract locals for `marker.infoWindow.title` and `...snippet`, at which point you wouldn't need the `!`

Reply:
Yep, good call!